x-pack/plugins/security_solution/common/api/timeline/export_timelines/export_timelines_route_schema.yaml

```diff
@@ -0,0 +1,48 @@
+openapi: 3.0.0
+info:
+  title: Elastic Security - Timeline - Import Timelines API
+  version: 8.9.0
+externalDocs:
+  url: https://www.elastic.co/guide/en/security/current/timeline-api-import.html
+  description: Documentation
+servers:
+  - url: 'http://{kibana_host}:{port}'
+    variables:
+      kibana_host:
+        default: localhost
+      port:
+        default: '5601'
+paths:
+  /api/timeline/_export:
+    post:
+      operationId: exportTimelines
+      description: Exports timelines as an NDJSON file
+      tags:
+        - access:securitySolution
+      parameters:
+        - in: query
+          name: file_name
+          schema:
+            type: string
+          description: The name of the file to export
+      requestBody:
+        description: The id of the timelines to export
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                ids:
+                  nullable: true
+                  type: array
+                  items:
+                    type: string
+      responses:
+        200:
```

Comment:
It looks like `POST /api/timelines/_export` has more than 1 return codes as mentioned below:

https://github.com/elastic/kibana/blob/cb444bc319007c6fa16e471b0c9ab5d1195f6420/x-pack/plugins/security_solution/server/lib/timeline/routes/timelines/export_timelines/index.ts#L48-L56

Reply:
Yep, updated. Thanks!